tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature

```diff
@@ -377,14 +402,18 @@ Feature: sharing
     And group "grp1" has been created
     And user "Brian" has been added to group "grp1"
     And user "Alice" has uploaded file with content "Random data" to "/randomfile.txt"
-    And user "Alice" has shared file "randomfile.txt" with group "grp1"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | randomfile.txt |
+      | space           | Personal       |
+      | sharee          | grp1           |
+      | shareType       | group          |
+      | permissionsRole | Viewer         |
```

Comment:
the scenario title and the steps do not match:
1. group name must have a capital letter
2. share should be an action (When step)

Reply:
1 should be invalid. it says username should be capital, not the group name.